include/prereq-build.mk

```diff
@@ -141,10 +141,19 @@ $(eval $(call SetupHostCommand,wget,Please install GNU 'wget', \
 $(eval $(call SetupHostCommand,perl,Please install Perl 5.x, \
 	perl --version | grep "perl.*v5"))
 
-$(eval $(call SetupHostCommand,python,Please install Python 2.x, \
-	python2.7 -V 2>&1 | grep Python, \
-	python2 -V 2>&1 | grep Python, \
-	python -V 2>&1 | grep Python))
+$(eval $(call CleanupPython2))
+
+$(eval $(call SetupHostCommand,python,Please install Python >= 3.5, \
+	python3.7 -V 2>&1 | grep -E 'Python 3\.[5-9]\.?', \
+	python3.6 -V 2>&1 | grep -E 'Python 3\.[5-9]\.?', \
+	python3.5 -V 2>&1 | grep -E 'Python 3\.[5-9]\.?', \
+	python3 -V 2>&1 | grep -E 'Python 3\.[5-9]\.?'))
```

Comment:
This could cause problems when python 3.10 is used in the future.

Reply:
I think we can worry about it when/if that time comes